torch/csrc/api/include/torch/nn/cloneable.h

```diff
@@ -58,9 +58,7 @@ class Cloneable : public virtual Module {
         copy->parameters_[parameter.key].copy_(
             *parameter, /*non_blocking=*/true);
       } else {
-        at::detail::set_data(
-            copy->parameters_[parameter.key],
-            autograd::Variable(*parameter).data().clone());
+        copy->parameters_[parameter.key].set_data(autograd::Variable(*parameter).data().clone());
```

Comment:
This also changes the variable type, like `detail::set_data` used to? Otherwise LGTM, although `set_data` is more "public" than `at::detail::set_data` was intended to be (!)

Reply:
I'm fine with prefixing it with an underscore to make it more private. The primary matter at hand was I didn't want to go through the trouble of friending this function.